src/dmd/expression.d

```diff
@@ -642,6 +642,7 @@ extern (C++) abstract class Expression : ASTNode
     ubyte parens;   // if this is a parenthesized expression
     Type type;      // !=null means that semantic() has been run
     Loc loc;        // file location
+    ubyte inuse;    // detect expressionSemantic() loops
```

Comment:
Please open a PR to update the `.h` file.

Reply:
damn again... yes coming soon